wpilibj/src/main/java/edu/wpi/first/wpilibj2/command/SynchronousPIDSubsystem.java

```diff
@@ -0,0 +1,79 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj2.command;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+
+import static edu.wpi.first.wpilibj.util.ErrorMessages.requireNonNullParam;
+
+/**
+ * A subsystem that uses a {@link PIDController} to control an output.  The controller is run
+ * synchronously from the subsystem's periodic() method.
+ */
+public abstract class SynchronousPIDSubsystem extends SendableSubsystemBase {
```

Comment:
Should we just call this PIDSubsystem?  Since there's no async one...

Reply:
Yeah, I was wondering about that.  On the one hand, it's a lot less verbose.  On the other hand, naming it this alerts people that there's been a significant change in functionality, and allows us to add an asynchronous one later if we really want to (I doubt we will, but you never know).

I could go either way on this, so I'll do whatever everyone else thinks is best.

Reply:
Upon reflection, the decrease in verbosity clearly outweighs everything else; I've made the change.